recipes/tl/all/conanfile.py

```diff
@@ -26,18 +29,22 @@ def configure(self):
             "Visual Studio": "15"
         }
 
+        def lazy_lt_compiler_version(version1, version2):
+            v1, *v1_res = version1.split(".", 1)
+            v2, *v2_res = version2.split(".", 1)
+            if v1 == v2 and v1_res and v2_res:
+                return lazy_lt_compiler_version(v1_res, v2_res)
+            return v1 < v2
```

Comment:
Can this be replaced with the following
```python
version1.split(".") < version2.split(".")
```

Reply:
No:

```
>>> "1.7".split(".") < "1.7.3".split(".")
True
```

Reply:
Indeed.

How about this one?

```python
def lazy_lt(v1, v2):
  lv1 = v1.split(".")
  lv2 = v2.split(".")
  lv1 = lv1 + ["0"] * (max((len(lv1), len(lv2))) - len(lv1))
  lv2 = lv2 + ["0"] * (max((len(lv1), len(lv2))) - len(lv2))
  return lv1 < lv2
```

Reply:
Probably not.

But this works:

```python
def lazy_lt_version(v1, v2):
    lv1 = v1.split(".")
    lv2 = v2.split(".")
    min_length = min(len(lv1), len(lv2))
    return lv1[:min_length] < lv2[:min_length]
```

Reply:
Yours does not compute `lazy_lt_version("1.2", "1.2.3")` correctly.

Reply:
I want 1.2 < 1.2.3 to be False

Reply:
How about `lazy_lt_version("1.2.3", "1.2")` then?

Reply:
False also

Reply:
Ok then. I guess I don't fully understand your intention.

Reply:
My intention is to not raise if compiler.version is 6 for gcc, because min version is gcc 6.4

Reply:
Ooh, that makes total sense.